Documentation/ExtensionConfiguration/Index.rst

```diff
@@ -15,35 +15,40 @@ advise) the extension needs to contain its own :file:`composer.json`.
 
 .. include:: /CodeSnippets/ExtensionConfiguration/ComposerJson.rst.txt
 
-For historic reasons TYPO3 extension names have to be written in lower case
-separated by underscores. We suggest to use the extension key for the directory
-of the extension as well to minimize confusion. So the extension in the path
-:file:`site_package` has to have the
-same "extension-key" to be defined in the "extra" section of the composer.json.
+For historic reasons TYPO3 extension names are written in
+lower case words and separated by underscores if there are more than one. This
```

Comment:
"if there are more than one" - why this addition?

Reply:
Hi Chris I wanted to stress that extension names can be 1 word or they can be more than 1 word. Maybe I wasn't very clear